test/mlir/driver/llvm.ident.mlir

```diff
@@ -3,7 +3,7 @@
 // RUN: cat %t.ll | FileCheck %s
 
 // CHECK: !llvm.ident = !{![[MD:[0-9]*]]}
-// CHECK: ![[MD]] = !{!"onnx-mlir version 1.0.0 ({{.*}}/onnx-mlir{{.*}} {{[a-zA-Z0-9]+}} {{.*}}/llvm-project{{.*}} {{[a-zA-Z0-9]+}})"}
+// CHECK: ![[MD]] = !{!"onnx-mlir version 1.0.0 ({{.*}}/onnx-mlir{{.*}} {{[a-zA-Z0-9]+}}, {{.*}}/llvm-project{{.*}} {{[a-zA-Z0-9]+}})"}
```

Comment:
Will this test pass if the vendor is supplied?

Reply:
That's a very good question, this test would fail if the vendor-specific text is supplied. 
Not sure what to do about that, may have to remove this test case. Any suggestions?

Reply:
You can do a couple of things: modify the test so that it passed regardless - this could mean removing parts of the verification string, or adding all of the information to the version regardless (e.g. even if the vendor string is supplied maybe we still want the commits). The other thing you could do is include the test based on whether a vendor string was supplied.

Reply:
When vendor-specific text is supplied, this test case will fail expectedly (XFAIL) now.